src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -500,6 +500,7 @@ def basic(
             analyzer_env=analyzer_env,
             tools=tools,
             task_env=task_env,
+            target_timeout=target_timeout,
```

Comment:
you also need to fix the invocation on line 163. `target_timeout` is missing from the list of argument there too

Reply:
I think this is okay: the LibFuzzerFuzz task definition doesn't support timeout (it doesn't have the required feature).